[2024-10-02T19:02:00.621+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T19:02:00.643+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-05-10T00:00:00+00:00 [queued]>
[2024-10-02T19:02:00.860+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-05-10T00:00:00+00:00 [queued]>
[2024-10-02T19:02:00.860+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T19:02:00.878+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): carregar> on 2024-05-10 00:00:00+00:00
[2024-10-02T19:02:00.885+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=7251) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T19:02:00.887+0000] {standard_task_runner.py:63} INFO - Started process 7261 to run task
[2024-10-02T19:02:00.887+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'carregar', 'scheduled__2024-05-10T00:00:00+00:00', '--job-id', '1615', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp6lefy3iv']
[2024-10-02T19:02:00.889+0000] {standard_task_runner.py:91} INFO - Job 1615: Subtask carregar
[2024-10-02T19:02:00.933+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-05-10T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T19:02:01.023+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='carregar' AIRFLOW_CTX_EXECUTION_DATE='2024-05-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-10T00:00:00+00:00'
[2024-10-02T19:02:01.027+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T19:02:01.082+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T19:02:01.083+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T19:02:01.091+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=carregar, run_id=scheduled__2024-05-10T00:00:00+00:00, execution_date=20240510T000000, start_date=20241002T190200, end_date=20241002T190201
[2024-10-02T19:02:01.141+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T19:02:01.156+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T19:02:01.159+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
